docs/requirements.rst

```diff
@@ -47,6 +47,14 @@ Usage
     set()
     >>> url_req.marker
     <Marker('os_name == "a"')>
+    >>> # You can do simple comparisons between requirement objects:
+    >>> Requirement("packaging") == Requirement("packaging")
+    True
+    >>> # You can also do simple comparissons between sets of requirements:
```

Comment:
```suggestion
    >>> # You can also perform simple comparisons between sets of requirements:
```

Reply:
Thank you very much for the suggestion @pradyunsg.